Source/Blazorise/Utilities/ClassBuilder.cs

```diff
@@ -94,7 +94,7 @@ public string Class
             {
                 if ( dirty )
                 {
-                    builder = new();
+                    builder.Clear();
```

Comment:
Good optimizations. According to the online benchmarks, it seems this is a little faster than `new().` To be honest I didn't know about it.